### PR TITLE
Add JSON-LD context and validation info to Serialization chapter

### LINKED ISSUE #998: Document the JSON-LD context
The published context should be documented (contents, location), etc.

This should probably go in the "Serializations" chapter.

### PR DESCRIPTION
**Pull request by bact**
Use info from

- https://github.com/spdx/spdx-3-model/blob/main/serialization/README.md
- https://github.com/spdx/spdx-3-model/blob/main/serialization/jsonld.md

This will resolve #998

Please also check the 3 URLs used in the content if they are the preferred location:

1. JSON context: https://spdx.org/rdf/3.0.1/spdx-context.jsonld
1. OWL ontology: https://spdx.org/rdf/3.0.1/spdx-model.ttl
1. JSON Schema: https://spdx.org/schema/3.0.1/spdx-json-schema.json

----

Note: Those 3.0.1 URLs are based on 3.0.0 URLs:

1. JSON context: https://spdx.org/rdf/3.0.1/spdx-context.jsonld
    - Based on this 3.0 URL:
    - https://github.com/spdx/spdx-spec/blob/b2f746495fad574507c881b54d557097aaf44029/examples/jsonld/package_sbom.json#L2
    - https://github.com/spdx/spdx-spec/blob/b2f746495fad574507c881b54d557097aaf44029/bin/check-examples.sh#L60
    - Already used here: 
    - https://github.com/spdx/spdx-spec/blob/b2f746495fad574507c881b54d557097aaf44029/docs/annexes/rdf-model.md?plain=1#L7 
1. OWL ontology: https://spdx.org/rdf/3.0.1/spdx-model.ttl
    - Based on this 3.0.0 URL:
    - https://github.com/spdx/spdx-spec/blob/b2f746495fad574507c881b54d557097aaf44029/bin/check-examples.sh#L20-L22 
1. JSON Schema: https://spdx.org/schema/3.0.1/spdx-json-schema.json
    - Based on this 3.0.0 URL:
    - https://github.com/spdx/spdx-spec/blob/b2f746495fad574507c881b54d557097aaf44029/bin/check-examples.sh#L15-L17

### CODE REVIEW

File: docs/serializations.md [hunk 1]
```diff
@@ -81,11 +81,58 @@ more serialized forms of itself.
 
 When serializing a physical SpdxDocument, any property of the logical element
 that can be natively represented within the chosen serialization format
-(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) may utilize
+(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) MAY utilize
```
Comment (goneall) [inline docs/serializations.md:84]: I don't know if we need the emphasis of capitalization on this or the next 2 changes - but I don't feel strongly if others prefer caps.

File: docs/serializations.md [hunk 2]
```diff
@@ -81,11 +81,58 @@ more serialized forms of itself.
...
+### JSON-LD context file
+
+JSON-LD contexts allow JSON documents to use simple, human-readable, locally
+defined terms while ensuring data interoperability across different systems.
+
+The SPDX global JSON-LD context file MUST be used universally for all SPDX
+documents in JSON-LD format that adhere to a specific SPDX version.
+
+SPDX global JSON-LD context file is available at:
+[https://spdx.org/rdf/3.0.1/spdx-context.jsonld](https://spdx.org/rdf/3.0.1/spdx-context.jsonld)
```
Comment (goneall) [inline docs/serializations.md:104]: Just a note: This link is currently not live - if I can get the generated schema and context files, I can update the site and make them available.

**Review by goneall — APPROVED** [review]
Overall, looks good - one minor suggestion on use of caps, but I'll approve.  If @zvr and/or @kestewart approve of the current state, please feel free to merge.

I'll need to add the context, ontology, and schema to the spdx.org website for the links in this PR to be live.

File: docs/serializations.md (hunk 1 above)
Comment (bact) [reply docs/serializations.md]: I put it lowercase now.

**Review by bobmartin3000 — APPROVED** [review]
LGTM

**Comment by bact** [comment]
> I'll need to add the context, ontology, and schema to the spdx.org website for the links in this PR to be live.

Keep track with https://github.com/spdx/spdx-spec/issues/1060

**Review by zvr — APPROVED** [review]

**Pull request merged**